benchmarks/deconvolution-1D/server.py

```diff
@@ -0,0 +1,116 @@
+import umbridge
+import numpy as np
+import cuqi
+
+
+class Deconvolution1D(umbridge.Model):
+    """Base benchmark for all 1D deconvolution problems"""
+
+    dim = 128  # Dimension of the 1D signal
+
+    def __init__(self, name):
+        """Initialize the model
+
+        parameters
+        ----------
+        name : str
+            Name of the model
+
+        """
+        # Load data from file
+        data = np.load("data_pc.npz")
+
+        # Set up test problem
+        self.TP = cuqi.testproblem.Deconvolution1D(
```

Comment:
I am trying to follow the motivation for using the testproblem here, for example specifying both a phantom and separately observed data, which may or may not arise from the same phantom. That seems redundant/a source of confusion/misunderstanding. As far as I can see the user has no obvious way of knowing how the data was generated. How will we as developers remember with which parameters this data file was generated? I believe it either 1) be made more explicit in the code (not just in a  regression test) how that data file was generated (phantom, model, noise, seed etc) or 2) generated by the file at runtime instead of being pregenerated and loaded.  2) may be problematic though when used as a (fixed) benchmark, since how will we know that exactly the same observed data is generated on separate runs of the code say 3 years apart on different architectures.

Reply:
just my 2 cents, might be an idea to generate reference data in the dockerfile (so while building, not running the model)?

Reply:
I made how the data was generated more clear